src/protocol/thread/node.ts

```diff
@@ -106,8 +106,7 @@ export class NodeFront {
   }
 
   get doctypeString() {
-    // NYI
-    return "unknown";
+    return this.nodeName;
```

Comment:
We can probably actually simplify it a bit more by 

```suggestion
    return `<!DOCTYPE ${this.nodeName}>`
```